WordPress/src/main/java/org/wordpress/android/WordPress.java

```diff
@@ -446,6 +438,15 @@ public static Blog setCurrentBlog(int id) {
         return currentBlog;
     }
 
+    public static Blog setCurrentBlogAndSetVisible(int id) {
+        Blog blog = getBlog(id);
```

Comment:
> I also noticed that setCurrentBlog is very similar to getBlog.

Good catch! Didn't notice that. In fact, `setBlog()` returning a value didn't make sense so I looked at its usages and no one used the return value so I removed it.

>  It's probably better to have setCurrentBlogAndSetVisible calling setCurrentBlog and in order, change setCurrentBlog that should call getBlog.

So it was calling it before, but as the return value. Now, it calls it first and then sets the blog as visible. I believe that's what you meant.